total_tolles_ferleihsystem/src/app/webapp-settings.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WebappSettingsService {
+  constructor(private readonly http: HttpClient) { }
+
+  getWebappSettings(): Observable<WebappSettings> {
+    return this.http.get<WebappSettings>('webappSettings.json');
+  }
+}
+
+/**
+ * Immutable representation of the settings configured in webappSettings.json
+ */
+export class WebappSettings {
```

Comment:
this class only claims to be immutable but is not. This could lead to some bad errors when someone tries to change values anyway...
Typescript knows a readonly access modifier (see constructor above)